main.c

```diff
@@ -101,6 +101,7 @@ void main(void){
     vector_init(&tests_to_run);
     load_conf_file("D:\\config.txt");
     open_output_file("D:\\kernel_tests.log");
+    srand(NULL);
```

Comment:
Ideally this seed would be chosen by the config but defaults to something known (like `0`).